configure.ac

```diff
@@ -40,6 +54,11 @@ AC_CHECK_FUNCS([memset strchr])
 
 AC_C_BIGENDIAN
 
+# Attempt to use c++11
+AX_CXX_COMPILE_STDCXX(11, noext, optional)
```

Comment:
This macro effectively tries to add '-std=c++11' to CXXFLAGS. If unsuccessful, nothing is changed.